hepa-uv/firmware/led_hardware.c

```diff
@@ -0,0 +1,494 @@
+/**PB11 DECK_LED DRIVE
```

Comment:
we should update or remove these comments to correct pin mappings

Reply:
removed the pin mapping, will add in separate pr.